src/bp/nlu-server/model/model-service.ts

```diff
@@ -0,0 +1,48 @@
+import { NLU } from 'botpress/sdk'
+import _ from 'lodash'
+import LRUCache from 'lru-cache'
+
+import ModelRepository from './model-repo'
+
+export default class ModelService {
+  private _modelRepo: ModelRepository
+  private _modelCache = new LRUCache<string, NLU.Model>({
+    max: 500000000,
```

Comment:
500,000,000 characters of one bute each = 500 Mb right?

Reply:
Elsewhere we try to add a comment on how you compute that . an utf-8 char is up to 4 bytes